_includes/v22.2/backups/updated-backup-privileges.md

```diff
@@ -0,0 +1,17 @@
+{{site.data.alerts.callout_info}}
+From v22.2, CockroachDB supports a backup privilege model that provides finer control over a user's privilege to take backups. There is continued support for the [existing privilege model](#existing-required-privileges) in v22.2. However, this will be removed in v23.1.
```

Comment:
We should probably not commit to removing in 23.1 cc: @livlobo. We can be a little vague and say removed in a future release?